datumaro/plugins/coco_format/extractor.py

```diff
@@ -300,26 +369,20 @@ def _load_annotations(self, ann, image_info=None, parsed_annotations=None):
                             )
                     else:
                         # merge all parts into a single mask RLE
-                        rle = self._lazy_merged_mask(
-                            segmentation, image_info["height"], image_info["width"]
-                        )
+                        img_h = self._parse_field(image_info, "height", (int, float))
+                        img_w = self._parse_field(image_info, "width", (int, float))
```

Comment:
I don't think `height` and `width` should be allowed to be `float`s.

Reply:
Ok, agree. I thought, it can be useful to allow values like 100.0, but it seems like not a good idea.